client/blocks/reader-email-settings/docs/example.jsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ReaderEmailSettings from 'blocks/reader-email-settings';
+import Card from 'components/card';
+import SyncReaderFollows from 'components/data/sync-reader-follows';
+
+const exampleSiteId = 70135762; // Longreads
+
+export default class ReaderEmailSettingsExample extends React.Component {
+	static displayName = 'ReaderEmailSettings';
+
+	render() {
+		return (
+			<Card>
+				<SyncReaderFollows key="syncReaderFollows" />
```

Comment:
I'm just mirroring the other use in `reader-main`, which also probably doesn't need it :)